appendices/assignments.md

```diff
@@ -0,0 +1,197 @@
+# Course Assignments
+
+<!-- Perspective for this section would be on the student, but the instructor
+section would be of course on the instructor. Or should this be entirely 
+targeted to the instructor? -->
+
+This course is designed around completing at least two project assignments.
+You would largely work on these assignments outside of class time.
+Each project assignment has several milestones you would need to complete
+throughout the course, usually due shortly after encountering them during
+the course. The project assignments are to:
+
+1. Create a small and simple package that you would complete on your own.
+2. Make small and simple improvements to existing packages on GitHub that you
+would complete on your own.
```

Comment:
I like @cwickham's suggestion and I think we can also get rid of the "would complete on your own" bit. 

```suggestion
2. Make small and simple improvements to a package on GitHub.
```